managing-os/templates/fedora-minimal.md

```diff
@@ -65,11 +65,30 @@ Use case | Description | Required steps
 **USB** | If you want USB input forwarding to use this template as the basis for a [USB](/doc/usb/) qube such as `sys-usb` | Install `qubes-input-proxy-sender`
 **VPN** | You can use this template as basis for a [VPN](/doc/vpn/) qube | Use the `dnf search "NetworkManager VPN plugin"` command to look up the VPN packages you need, based on the VPN technology you'll be using, and install them. Some GNOME related packages may be needed as well. After creation of a machine based on this template, follow the [VPN howto](/doc/vpn/#set-up-a-proxyvm-as-a-vpn-gateway-using-networkmanager) to configure it.
 **DVM Template** | If you want to use this VM as a [DVM Template](/doc/glossary/#dvm-template) | Install `perl-Encode`
-  
-  
-
+ 
 A comprehensive guide to customizing the minimal template is available [here][GUIDE]
 
+Qubes 4.0, fedora-26
+------
+In Qubes4.0 the `fedora-26-minimal` template is available and can be installed issuing the following command:
+
+~~~
+[user@dom0 ~]$ sudo qubes-dom0-update qubes-template-fedora-25-minimal
+~~~
+
+This template is slightly bigger (~600MB). All the suggestions made in the previous section apply. In addiction to this, in many situations packages from the qubes-core-agent suite will be needed to make the customized minimal template to work properly. These packages are:
```

Comment:
Interesting. @mossy-nw claims it's at least 2GB in https://github.com/QubesOS/qubes-doc/pull/507.

Reply:
It looks like compressed/uncompressed difference. RPM package is about 650MB, which results in about 2GB disk usage after installation.

Reply:
Yup. 600MB is the amount of stuff dnf has to download. it reaches ~1.5GB when installed, if I remember well.

Reply:
Thanks for following up on this -- in R3.2 Qubes manager reports ~2680MiB, which agrees with `[dom0]$ du` -- `private.img` alone is 2.0G.  I don't want to needlessly nitpick, I just want the docs to be clear for folks using, say, older hardware with a 120GB SSD.  Finally, I ran `[dom0]$ qvm-trim-template`, just to see, and it's now 1.7G.  Should folks with limited disk space be trimming templates?

Reply:
Yes.